chapters/data-analysis.tex

```diff
@@ -1,292 +1,488 @@
+
+Analysis files should be as simple as possible, so you can focus on the econometrics.
+All research decisions should be made very explicit in the code.
+This includes clustering, sampling, and control variables, to name a few. 
+If you have multiple analysis data sets, each of them should have a descriptive name about its sample and unit of observation.
+As your team comes to a decision about model specification, you can create globals or objects in the master script to use across scripts.
+This is a good way to make sure specifications are consistent throughout the analysis. It's also very dynamic, making it easy to update all scripts if needed.
+It is completely acceptable to have folders for each task, and compartmentalize each analysis as much as needed.
+It is always better to have more code files open than to keep scrolling inside a given file.
 
-\section{Visualizing data}
-
-\textbf{Data visualization}\sidenote{\url{https://dimewiki.worldbank.org/wiki/Data_visualization}} is increasingly popular,
-\index{data visualization}
-but a great deal of it is very low quality.\cite{healy2018data,wilke2019fundamentals}
-The default graphics settings in Stata, for example,
-are pretty bad.\sidenote{Gray Kimbrough's
-\textit{Uncluttered Stata Graphs} code is an excellent
-default replacement that is easy to install.
-\url{https://graykimbrough.github.io/uncluttered-stata-graphs/}}
-Thankfully, graphics tools like Stata are highly customizable.
-There is a fair amount of learning curve associated with
-extremely-fine-grained adjustment,
-but it is well worth reviewing the graphics manual
-to understand how to apply basic colors, alignments, and labelling tools\sidenote{
-\url{https://www.stata.com/manuals/g.pdf}}
-The manual is worth skimming in full, as it provides
-many visual examples and corresponding code examples
-that detail how to produce them.
-
-Graphics are hard to get right because you, as the analyst,
-will already have a pretty good idea of what you are trying to convey.
-Someone seeing the illustration for the first time,
-by contrast, will have no idea what they are looking at.
-Therefore, a visual image has to be compelling in the sense
-that it shows that there is something interesting going on
-and compels the reader to figure out exactly what it is.
-A variety of resources can help you
-figure out how best to structure a given visual.
-The \textbf{Stata Visual Library}\sidenote{
-\url{https://worldbank.github.io/Stata-IE-Visual-Library/}}
- has many examples of figures that communicate effectively.
+To accomplish this, you will need to make sure that you have an effective data management system, including naming, file organization, and version control.
+Just like you did with each of the analysis datasets, name each of the individual analysis files descriptively.
+Code files such as \path{spatial-diff-in-diff.do}, \path{matching-villages.do}, and \path{summary-statistics.do} 
```

Comment:
Very Stata specific examples? Can we make them more agnostic or include other less Stata specific examples?

Reply:
lol sure change one to `.r` then ;-)

Reply:
.R and .py in 83a294a